api/main.py

```diff
@@ -308,3 +326,41 @@ async def publish(raw: dict, channel: str, user: User = Depends(get_user)):
     attributes = dict(raw)
     data = attributes.pop('data')
     await pubsub.publish_cloudevent(channel, data, attributes)
+
+
+# -----------------------------------------------------------------------------
+# Regression
+
+@app.post('/regression')
```

Comment:
Note: This means we can create or update regression objects with the `/regression` endpoint but only get them with the `/nodes` endpoint. I think this is fine and as far as I know there is no general expectation for an API to provide a `GET` for every endpoint. If we needed to implement it later we could easily do that by using the same logic as in the `/node` `GET` handler but with a hard-coded value for `kind`.

Reply:
Fair enough.